include/boost/unordered/detail/foa/table.hpp

```diff
@@ -527,22 +544,34 @@ class table:table_core_impl<TypePolicy,Hash,Pred,Allocator>
   friend bool operator!=(const table& x,const table& y){return !(x==y);}
 
 private:
-  template<typename ExclusiveLockGuard>
-  table(compatible_concurrent_table&& x,ExclusiveLockGuard):
+  template<typename ArraysType,typename ExclusiveLockGuard>
+  table(
+    compatible_concurrent_table&& x,
+    arrays_holder<ArraysType,Allocator>&& ah,ExclusiveLockGuard&):
```

Comment:
Yeah, I wasn't sure with this one. Because we were delegating and only really using this constructor in one context, I figured it couldn't hurt to convey that we were still using the lock guard but you're right, we can just drop it entirely.